compiler/AST/AstToText.cpp

```diff
@@ -970,6 +978,96 @@ void AstToText::appendExpr(CallExpr* expr, bool printingType)
 
         }
 
+        // mText += std::to_string(expr->numActuals());
+        // mText += ' ';
+        // appendExpr(expr->get(1), printingType);
+        // mText += ' ';
+        // appendExpr(expr->get(2), printingType);
+        // mText += ' ';
+        // appendExpr(expr->get(3), printingType);
+        else if (expr->numActuals() == 3)
```

Comment:
I think this might be punting the problem.  It's possible for domains to be declared with a different stridable setting, parSafe settings, index type, rank, etc.  There's a few combinations, it isn't just limited to 2 or 3 arguments.  While this is certainly handling cases we didn't handle before, I think we can do even better and rearrange this to appropriately handle any combination.  What do you think?

Reply:
Yeah, I guess we can do handle many cases than before as however we have to changing it to 3 arguments. To be honest I didn't go through all the `domain` calls. I'll go through them once.